docs/user/tutorials/01-10-setup-custom-domain-for-workload.md

```diff
@@ -7,32 +7,55 @@ This tutorial shows how to set up a custom domain and prepare a certificate requ
-* If you use a cluster not managed by Gardener, install the [External DNS Management](https://github.com/gardener/external-dns-management#quick-start) and [Certificate Management](https://github.com/gardener/cert-management) components manually in a dedicated namespace.
+* [Deploy a sample HTTPBin Service](./01-00-create-workload.md).
+* If you use a cluster not managed by Gardener, install the [External DNS Management](https://github.com/gardener/external-dns-management#quick-start) and [Certificate Management](https://github.com/gardener/cert-management) components manually in a dedicated namespace. SAP BTP, Kyma runtime clusters are managed by Gardener so you are not required to install any additional components.
 
 ## Steps
 
-1. Create a Secret containing credentials for the DNS cloud service provider account in your namespace.
+1. Create a Secret containing credentials for the DNS cloud service provider account in your namespace. To learn how to do it, follow the [External DNS Management guidelines](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management).
 
-    * Choose your DNS cloud service provider and create a Secret in your namespace. To learn how to do it, follow [the guidelines](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management) provided in the External DNS Management documentation. 
-    * Export the name of the created Secret as an environment variable:
+<!-- tabs:start -->
+#### **Kyma Dashboard**
 
-      ```bash
-      export SECRET={SECRET_NAME}
-      ```
+1. Select the namespace you want to use.
+2. Go to **Configuration > Secrets**.
+3. Select **Create Secret** and provide your configuration details.
+4. Select **Create**.
 
-2. Create a DNSProvider custom resource (CR).
+#### **kubectl**
+Use `kubectl apply` to create a Secret containing the credentials and export its name as an environment variable:
 
-    * Export the following values as environment variables. 
-      > [!NOTE]
-      > As `SPEC_TYPE`, use the relevant provider type. The `DOMAIN_NAME` value specifies the name of a domain that you own, for example, `mydomain.com`. 
+```bash
+export SECRET={SECRET_NAME}
+```
+<!-- tabs:end -->
+
+2. Create a DNSProvider custom resource (CR).
+    
+<!-- tabs:start -->
+  #### **Kyma Dashboard**
+
+  1. Go to **Configuration > DNS Providers**.
+  2. Select **Create DNS Provider**, switch to the `Advanced` tab, and provide the details:
+    - **Name**: `dns-provider`
+    - **Type**: is the type of your DNS cloud service provider.
+    - Add the annotation **dns.gardener.cloud/class**: `garden`
+    - In the `Secret Reference` section, add these fields:
+        - **Namespace**: is the name of the namespace in which you created the Secret containing the credentials. 
+        - **Name**: is the name of the Secret.
+    - In the `Include Domains` section, add the field:
+      - **Include Domains**: is the name of your custom domain.
```

Comment:
Some of the bullet points do not render correctly. Please improve the indentation.